tests/storage_test.go

```diff
@@ -195,22 +199,39 @@ var _ = Describe("Storage", func() {
 
 		Context("[rfe_id:3106][crit:medium][vendor:cnv-qe@redhat.com][level:component]With ephemeral alpine PVC", func() {
 			// The following case is mostly similar to the alpine PVC test above, except using different VirtualMachineInstance.
+			var isRunOnKindInfra bool
+			tests.BeforeAll(func() {
+				isRunOnKindInfra = tests.IsRunningOnKindInfra()
+			})
+
 			It("[test_id:3136]should be successfully started", func() {
+				tests.SkipPVCTestIfRunnigOnKindInfra()
+
 				// Start the VirtualMachineInstance with the PVC attached
 				vmi := tests.NewRandomVMIWithEphemeralPVC(tests.DiskAlpineHostPath)
-				tests.RunVMIAndExpectLaunch(vmi, 90)
-
+				if isRunOnKindInfra {
+					tests.RunVMIAndExpectLaunchIgnoreWarnings(vmi, 90)
```

Comment:
What is the exact warning you are ignoring?

Reply:
All info about this topic - along with the specific warning you're asking about - can be found in [this mailing list question](https://groups.google.com/forum/#!topic/kubevirt-dev/u8DL-8-KFxY).

Unfortunately, I had no responses (yet?).

Reply:
Thanks for replying on [the thread](https://groups.google.com/forum/#!topic/kubevirt-dev/u8DL-8-KFxY)!

I've pushed https://github.com/kubevirt/kubevirt/pull/3233 to address that, but it doesn't work for one of the tests - I'll need someone storage savy to help me out.

Could we 'get away with' opening an issue on kubevirt, linking against it on the test, and fix it afterwards ?